core/app/models/concerns/spree/user_methods.rb

```diff
@@ -23,6 +23,8 @@ module UserMethods
       has_many :store_credit_events, through: :store_credits
       money_methods :total_available_store_credit
 
+      has_many :wallet_payment_sources, foreign_key: 'user_id', class_name: 'Spree::WalletPaymentSource'
+
```

Comment:
Personally I'm not a huge fan of that module, and most of it is going away.  How about the reverse - bring the `has_many` from that module into here? Then the whole module goes away soon.